test/compilers/allows_import_in_dart_code_test.dart

```diff
@@ -9,7 +9,7 @@ import '../test_pub.dart';
 import '../serve/utils.dart';
```

Comment:
Nit: Directory names are usually singular in pub, so can you call the directory "compiler" instead of "compilers"?